scripts/build.js

```diff
@@ -25,22 +25,24 @@ const exec = (command, extraEnv) =>
     env: Object.assign({}, process.env, extraEnv),
   });
 
-const ignoreGlobs = ['**/__tests__/*', '**/*-test.js', '**/*-story.js'].join(
-  ','
-);
+const ignoreGlobs = [
+  `"**/__tests__/*"`,
```

Comment:
Would these need to be:

```
**/**/*-test.js
**/**/*-story.js
```

So that it's not just top-level directories?

Reply:
I believe the double wild card ** already allows for multiple directory levels (as opposed to the single *)